Mathlib/LinearAlgebra/AffineSpace/FiniteDimensional.lean

```diff
@@ -93,15 +93,14 @@ theorem finite_set_of_fin_dim_affineIndependent [FiniteDimensional k V] {s : Set
   @Set.toFinite _ s (finite_of_fin_dim_affineIndependent k hi)
 #align finite_set_of_fin_dim_affine_independent finite_set_of_fin_dim_affineIndependent
 
-open Classical
-
 variable {k}
 
 /-- The `vectorSpan` of a finite subset of an affinely independent
 family has dimension one less than its cardinality. -/
-theorem AffineIndependent.finrank_vectorSpan_image_finset {p : ι → P} (hi : AffineIndependent k p)
-    {s : Finset ι} {n : ℕ} (hc : Finset.card s = n + 1) :
+theorem AffineIndependent.finrank_vectorSpan_image_finset [DecidableEq P]
+    {p : ι → P} (hi : AffineIndependent k p) {s : Finset ι} {n : ℕ} (hc : Finset.card s = n + 1) :
     finrank k (vectorSpan k (s.image p : Set P)) = n := by
```

Comment:
Many lemmas in this file are missing the `[DecidableEq P]` needed to talk about `finset.image`